src/libponyc/codegen/codegen.c

```diff
@@ -882,7 +882,7 @@ bool codegen_init(pass_opt_t* opt)
   if(opt->features != NULL)
     opt->features = LLVMCreateMessage(opt->features);
   else
-    opt->features = LLVMCreateMessage("");
+    opt->features = LLVMGetHostCPUFeatures();
```

Comment:
@jemc I apologize that I wasn't clear. The logic I suggest was just high level pseudo-code and wasn't actual functioning C code.

I can get you the correct C code but that will take me some time to get right.